packages/search-token-lambda/README.md

```diff
@@ -0,0 +1,29 @@
+# Simple search token generation lamba function
+
+An [Netlify Function](https://functions.netlify.com/) to generate [Coveo search tokens](https://docs.coveo.com/en/1346/).
+
+## Setup environment
+
+Create the `.env` file at the root of this project using `.env.example` as starting point and make sure to replace all placeholder variables `<...>` by the proper information for your organization.
+For more involved configurations, you can modify the request parameters used in the `functions/token/generateToken.ts` file.
+
+## Installation
+
+```bash
+npm install
+```
+
+## Usage
+
+```bash
+npm start
+```
+
+This will start a server listening on port 9999. The app will return a [Coveo search token](https://docs.coveo.com/en/1346/) when you make a GET request to the [/.netlify/functions/token](http://localhost:9999/.netlify/functions/token) path. Every other path will respond by a **404 Not Found** error.
```

Comment:
```suggestion
This will start a server listening on port 9999. The app will return a [Coveo search token](https://docs.coveo.com/en/1346/) when you make a GET request to the [/.netlify/functions/token](http://localhost:9999/.netlify/functions/token) path. Every other path will respond with a **404 Not Found** error.
```